server/helper/handler.js

```diff
@@ -4,11 +4,45 @@ var helper = require('./helpers.js');
 
 module.exports = {
   getAllItems: function (req, res) {
-    item.getAll(function (items) {
-      res.send(items);
+    item.getAll(function (err, items) {
+      if(err) {return res.status(400);}
+      res.status(200).send(items);
```

Comment:
if send() method is called, status code is set to 200 automagically, so you don't need to explicitly call .status(200) here